test/it/uk/gov/hmrc/exports/connector/CustomsDataStoreConnectorSpec.scala

```diff
@@ -35,12 +33,13 @@ class CustomsDataStoreConnectorSpec extends IntegrationTestSpec {
 
   "CustomsDataStoreConnector.getEmailAddress" when {
 
-    "email service responds with OK (200)" should {
+    "email service responds with OK (200) and deliverable = true flag" should {
```

Comment:
"deliverable = true" isn't really accurate. Email service actually responds with a payload that does not contain the `undeliverable` json field.